addOns/oast/src/main/resources/org/zaproxy/addon/oast/resources/Messages.properties

```diff
@@ -60,6 +60,13 @@ oast.panel.table.column.referer=Referer
 oast.panel.currentState.tooltip.lastPolled = {0}: Last Polled Time
 oast.panel.currentState.lastPoll= {0}: {1}
 
+oast.popup.menu.insertPayload=Insert Payload
+oast.popup.menu.new=New
+oast.popup.menu.existing=Existing
+oast.popup.menu.boast=BOAST
+oast.popup.menu.interactsh=Interactsh
+oast.popup.menu.callback=Callback
```

Comment:
Should use the name of the services.

Reply:
I didnt understand can you please explain this a little more? do you mean like this ?
`oast.popup.menu.boast=Boast Service`

Reply:
Call `OastService#getName()` instead of adding new strings.

Reply:
okk thanks